docs/sql-guide/statements/statement-select.md

```diff
@@ -0,0 +1,186 @@
+* `expr` can be any constant, function or combination thereof joined by operators, or a subquery
+* Items in the select_list can be aliased with a column_alias
+* Any column referenced in a non-aggregated expression in the select_list must also appear in the group_by list
+
+### from_clause information
+
+![expr](/assets/images/sql-guide/from_clause.svg)
+![expr](/assets/images/sql-guide/table_or_subquery.svg)
+
+The table_or_subquery expression can be:
+* a table_name or table_valued_function
+* a parenthesized `SELECT` statement
+Both expressions can be aliased with a table_alias
+
+![expr](/assets/images/sql-guide/table_option.svg)
+
+The SHARDS option allows you to specify against with shards the query will run.
+
+### group_by_clause information
+
+![expr](/assets/images/sql-guide/group_by_clause.svg)
+
+`column_expr` specifies a column or non-aggregate calculation on a column which:
+* must exist in the from_clause
+* is not required to appear in the select_list
+
+A column must appear in the group_by_clause if it is referenced in a non-aggregated expression in the select_list
+
+### Wildcards
+
+Wildcards are used with the `LIKE` clause.
+
+* [SQL statement wildcards](https://www.w3schools.com/sql/sql_wildcards.asp){:target="_blank"}
+
+## Examples
+
+### SELECT statement with wildcard
+
+```sql
+SELECT * from services WHERE servicelist LIKE '%free%';
+```
+
+### SELECT COUNT
+```sql
+SELECT COUNT(*) FROM github-stats;
+```
+
+### SELECT TOP
+```sql
+SELECT TOP(10) * FROM github-stats;
+```
+
+### point select
+
+```sql
+select * from tbl where _id = 1
+select fld1, fld2 from tbl where _id = 1
+select _id, fld from tbl where _id = 1
+```
+
+### select distinct
+
+```sql
+select distinct fld from tbl
+```
+
+
+### select count
+
+```sql
+select count(*) from tbl
+select count(*) from tbl where fld = 1
+select count(*) from tbl where fld1 = 1 and fld2 = 2
+select count(distinct fld) from tbl
+```
+
+### select ids from segment
+```sql
+select _id from tbl where fld = 1
+select _id from tbl where fld in (1, 2)
+select _id from tbl where fld1 = 1 limit 1
+select _id from tbl where fld1 = 1 and fld2 = 2
+```
+
+### select int function
+
+```sql
+select min(fld) from tbl
+select max(fld) from tbl
+select sum(fld) from tbl
+select avg(fld) from tbl
+select min(fld) from tbl where fld = 1
+select max(fld) from tbl where fld = 1
+select sum(fld) from tbl where fld = 1
+select avg(fld) from tbl where fld = 1
+```
+
+### group by
+
+```sql
+select fld, count(*) from tbl group by fld
+select fld1, fld2, count(*) from tbl group by fld1, fld2
+select fld1, fld2, count(*) from tbl group by fld1, fld2 limit 1
+select fld1, fld2, count(*) from tbl where fld1 = 1 group by fld1, fld2
+select fld1, count(*) from tbl group by fld1 having count > 1
+select fld1, fld2, sum(fld3) from tbl where fld1 = 1 group by fld1, fld2
+select fld1, fld2, sum(fld3) from tbl where fld1 = 1 group by fld1, fld2 having count > 1
```

Comment:
having count(*) and for any other examples using having

Reply:
uh... not sure what you mean here?
Do you want me to add `having count(*)` on the end of all the statements?

Reply:
any of your examples that have "having count" need to be having count(*)